paymentsheet/src/main/java/com/stripe/android/paymentsheet/DisplayableSavedPaymentMethod.kt

```diff
@@ -5,5 +5,13 @@ import com.stripe.android.model.PaymentMethod
 internal data class DisplayableSavedPaymentMethod(
     val displayName: String,
     val paymentMethod: PaymentMethod,
-    val isCbcEligible: Boolean = false
-)
+    val isCbcEligible: Boolean = false,
+) {
+    fun isModifiable(): Boolean {
+        val hasMultipleNetworks = paymentMethod.card?.networks?.available?.let { available ->
+            available.size > 1
+        } ?: false
+
+        return isCbcEligible && hasMultipleNetworks
+    }
```

Comment:
moved this from `PaymentOptionsItem` to `DisplayableSavedPaymentMethod` so that we can share this logic between the SelectSavedPaymentMethods + ManageSavedPaymentMethods screens